pkg/auth/providers/saml/saml_client.go

```diff
@@ -420,7 +420,7 @@ func (s *Provider) HandleSamlAssertion(w http.ResponseWriter, r *http.Request, a
 				http.Redirect(w, r, redirectURL+"errorCode=500", http.StatusFound)
```

Comment:
Some line above has a comment "generate kubeconfig saml token" but should be encrypted token